src/Batch/Batch/Pools/NewBatchPoolCommand.cs

```diff
@@ -70,7 +70,9 @@ public class NewBatchPoolCommand : BatchObjectModelCmdletBase
 
         [Parameter]
         [ValidateNotNullOrEmpty]
-        public int? MaxTasksPerComputeNode { get; set; }
+        [GenericBreakingChange("MaxTasksPerComputeNode alias will be removed in an upcoming breaking change release", "4.2.0")]
```

Comment:
Please follow this example to announce breaking change for parameter: https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-attribute-help.md#a-parameter-is-being-deprecated